index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">      
+          <title> Tribute Page</title>
+          <link href="stylesheet"  rel="stylle.css">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <div id="main">
+            <h1 id="title">Dr. Norman Borlaug</h1>
```

Comment:
Kindly enclose the h1 within a header tag. This creates good accessibility and search engine optimization for your site